module/scene-config.js

```diff
@@ -0,0 +1,26 @@
+Hooks.on("renderSceneConfig", (sheet, html, data) => {
+    const document = sheet.object;
+
+    const campaignTension = game.settings.get("sgrpg", "campaignTension");
+    const sceneTension = data.document.getFlag("sgrpg", "sceneTensionDie") || "";
+
+    html.find(`select[name="journal"]`).parent().after(`\
+        <div class="form-group">
+            <label>Scene Tension Level</label>
+            <div class="form-fields">
+                <select type="text" name="flags.sgrpg.sceneTensionDie" data-dtype="String">
+                    <option value="" ${sceneTension ? "" : "selected"}>Campaign Base (${campaignTension})</option>
+                    <option value="d4" ${sceneTension === "d4" ? "selected" : ""}>Comedic (d4)</option>
+                    <option value="d6" ${sceneTension === "d6" ? "selected" : ""}>Standard (d6)</option>
+                    <option value="d8" ${sceneTension === "d8" ? "selected" : ""}>Growing (d8)</option>
+                    <option value="d10" ${sceneTension === "d10" ? "selected" : ""}>Dire (d10)</option>
+                    <option value="d12" ${sceneTension === "d12" ? "selected" : ""}>All is Lost (d12)</option>
```

Comment:
So finally i did get to it. It looks OK but i will (or you can if you have time) change this part so it shares settings with `game.settings.register -> choices`. There is sgrpg config into which this object (choices) would be moved and then used on both places to remove code and text duplication

Reply:
I'm afraid this would have to wait for at least a day in case I do it, as I have my own session to prepare.  

Do remember to keep the empty choice in the scene-config. Strictly speaking, they're not the same set of choices as the scene one has "Skip this and use the world setting" as a valid choice, when the world setting obviously lacks such a choice.

Reply:
So i will do it, no problem ;-)

Sure this "default" option will be added separately but rest is 100% same. Today i will probably merge and finish everything. Enjoy your session ;-)